src/RemoteSymbolProvider/StadiaSymbolStoreSymbolProviderTest.cpp

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+//#include <gmock/gmock.h>
```

Comment:
Unintended comment?

Reply:
😂  yes, that's a mistaking. Thanks for finding that